src/utils.rs

```diff
@@ -42,6 +42,48 @@ macro_rules! implement_enum_display {
     };
 }
 
+pub enum ValueOrRef<'a, T: Clone> {
```

Comment:
Can you add a note on what this does and/or what it's for?

Reply:
I dropped this.